stylesheets/cart.css

```diff
@@ -0,0 +1,46 @@
+.cart-container {
+    margin-top: 2.5rem;
+}
+
+.cart-header {
+    margin-top: 2rem;
+}
+
+.cart-horz-card {
+    max-width: 28rem;
+}
+
+.h-card-header {
+    gap: 0.5rem;
+}
+
+.cart-horz-card .badge-rating {
+    max-width: fit-content;
+    background-color: var(--green-500);
+    padding: 4px 8px;
+}
+
+.cart-product-info {
+    margin-top: 1.5rem;
+    width: 15rem;
+    
```

Comment:
remove space

Reply:
just did in the latest commit. Thanks for the suggestion